src/Components/Header/Header.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
-import './Header.css'
+import { Route }from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './Header.css';
+import SearchBar from '../SearchBar/SearchBar';
 
-const Header = () => {
+const Header = ({ filterMoviesDisplay }) => {
   return (
-    <h1 className="header">Rowdy Taters!</h1>
+    <header className="header">
+    <h1 className="header-title">Rowdy Taters!</h1>
+    <Route exact path='/' 
```

Comment:
Love this conditional rendering with Router. Super clean way to handle this logic.